docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -296,20 +283,20 @@ nodes` when run on the master.
 ### (Optional) Controlling your cluster from machines other than the master
 
 In order to get a kubectl on some other computer (e.g. laptop) to talk to your
-cluster, you need to copy the kubeconfig file from your master to your
-workstation like this:
+cluster, you need to copy the administrator kubeconfig file from your master
+to your workstation like this:
 
 ``` bash
 scp root@<master ip>:/etc/kubernetes/admin.conf .
```

Comment:
In https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/init.go#L53-L55 i see:
```bash
mkdir -p $HOME/.kube
sudo cp -i {{.KubeConfigPath}} $HOME/.kube/config
sudo chown $(id -u):$(id -g) $HOME/.kube/config
```